configs/bisenetv1/bisenetv1_r50-d32_lr5e3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py

```diff
@@ -0,0 +1,7 @@
+_base_ = './bisenetv1_r50-d32_lr5e3_4x4_512x512_160k_coco-stuff164k.py'
```

Comment:
Should check the config name, please refer to our config name manual.
It seems that the lr position is not correct.
5e3 --> 5e-3?